src/main/java/io/kestra/plugin/stripe/payment/CreateMethod.java

```diff
@@ -0,0 +1,131 @@
+package io.kestra.plugin.stripe.payment;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.stripe.model.PaymentMethod;
+import com.stripe.param.PaymentMethodCreateParams;
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.annotations.PluginProperty;
+import io.kestra.core.models.property.Property;
+import io.kestra.core.models.tasks.RunnableTask;
+import io.kestra.core.runners.RunContext;
+import io.kestra.plugin.stripe.AbstractStripe;
+import jakarta.validation.constraints.NotNull;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+import io.swagger.v3.oas.annotations.media.Schema;
+
+import java.util.Map;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Create a PaymentMethod in Stripe.",
+    description = "This task creates a PaymentMethod, typically with card details."
+)
+@Plugin(
+    examples = {
+        @Example(
+            title = "Create a card PaymentMethod",
+            full = true,
+            code = """
+                id: create_payment_method
+                namespace: company.team
+
+                tasks:
+                  - id: create_payment_method
+                    type: io.kestra.plugin.stripe.payment.CreateMethod
+                    apiKey: "{{ secret('STRIPE_API_KEY') }}"
+                    type: "card"
+                    cardNumber: "4242424242424242"
+                    expMonth: 12
+                    expYear: 2025
```

Comment:
And make it one String property to validate with "MM/YYYY" format please

```suggestion
                    expirationDate: "{{ secret('CREDIT_CARD_EXPIRATION') }}"
```